data/templated/example.xml

```diff
@@ -61,4 +61,16 @@
          <value>11</value>
       </values>
    </template>
+   <template name="GetItemRandom">
+      <query><![CDATA[SELECT COUNT(*) FROM item WHERE i_im_id = ?]]></query>
+      <types>
+         <type>RANDOM</type>
+      </types>
+      <values>
+         <value>binomial,0,9999</value>
+      </values>
+      <values>
+         <value>uniform,0,9999</value>
```

Comment:
Per commentary below, I think the representation here should be expanded to something more descriptive than a bar CSV.

For instance:
```xml
<types>
  <!-- maintain the actual parameter type here -->
  <type>integer</type>
</types>
<values>
  <!-- instead of specifying a <value> instance here, specify a new xml element - <value_distribution> -->
  <value_distribution type="binomial" min="0" max="9999" seed="42" />
  <!-- 
    add any other params needed for that distribution type in the properties of that value_distribution
    seed could be omitted and pulled from the global seed in that case, but this would allow individually selecting the starting seed for each parameter
  -->
</values>
```